doc/requirements.txt

```diff
@@ -18,3 +18,4 @@ tensornetwork==0.3
 toml
 torch==1.7.0+cpu
 torchvision==0.8.1+cpu
+orderedset==2.0.3
```

Comment:
This will need to be added to `setup.py` as well!

Reply:
Done.